src/routes/generator/stores.ts

```diff
@@ -1,8 +1,10 @@
 import { writable } from 'svelte/store';
 
 export const isMapModalOpen = writable(false);
+export const startModalOpen = writable(false);
 export const files = writable<FileList | null>(null);
 export const mapObjects = writable<
 	{ x: number; y: number; link: string; width: number; heigth: number }[]
 >([]);
-export const userProvidedLink = writable('');
+export const userProvidedLink = writable('Please provide the link for your image');
```

Comment:
Is this better suited to be in the placeholder text https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#placeholder

Reply:
added placeholder